data/core/terrain.cfg

```
@@ -3100,7 +3100,7 @@ Most units have about 60% defense in a castle."
     string=Mt
     editor_group=rough
     help_topic_text= _ "<italic>text='Mountains'</italic> are steep enough that units often have to climb over obstacles to move.
-By this nature, they provide a considerable defensive bonus for most troops, but they also severely impede any passage through them. Most cavalry simply cannot enter mountainous terrain; however, elvish cavalry is an exception to this, as are the goblin wolf riders. Both dwarves and trolls are native to mountainous terrain, and have a very easy time getting around.
+By this nature, they provide a considerable defensive bonus for most troops, but they also severely impede any passage through them. Units burdened by heavy armor, such as the Wesnoth army's cavalry and heavy infantry lines cannot even enter them. Aquatic races and creatures, such as the merfolk, with their piscine bodies rendering them unable to climb steep inclines, cannot enter mountains as well; however, nagas are an exception due to their many strong arms, rugged scales, and impressive strength. Both dwarves and trolls are native to mountainous terrain, and have a very easy time traversing them.
```

```suggestion
By this nature, they provide a considerable defensive bonus for most troops, but also severely impede any passage through them. Units burdened by heavy armor, such as the Wesnoth cavalry and heavy infantry lines, cannot even enter them. Aquatic races and creatures like the merfolk have piscine bodies which also render them unable to climb steep inclines. Nagas are an exception due to their strong arms and rugged scales. Both dwarves and trolls are native to mountainous terrain, and have an easy time traversing them.
```

It's a bit wordy right now, a little more concise would be nicer, and semicolons aren't to be used unless really necessary.